scripts/args.py

```diff
@@ -1,14 +1,25 @@
 #!/usr/bin/env python3
```

Comment:
i feel like this is an opportunity to delete some scripts that we never use :smile: